### DISCUSSION TITLE
The 'MonoAndroidDesignerPackage' package did not load correctly

### DISCUSSION THREAD

**Discussion by DzVibeCoder** [CommunityToolkit/Maui#127] [Q&A] [answered]
IDE: VISUAL STUDIO 2022 COMMUNITY PREVIEW 4.1
Project: .NET MAUI PREVIEW 8
i have a problem with Tools - Options - Xamarin - Android UI Designer :
- The 'MonoAndroidDesignerPackage' package did not load correctly
- an error occurred loading this property page
- The Toolbox is empty while trying ao add controls to the XAML File
- The Events doesn't load automatically even when pressing Enter Key+Space Key

**Comment by TheCodeTraveler** [comment]
Thanks for sending the video reproduction: https://www.youtube.com/watch?v=LFQ3MkYJWpE

Is this a .NET MAUI bug or a .NET MAUI Community Toolkit bug?

**Comment by DzVibeCoder** [comment, marked as answer]
The problem will be fixed in next preview of Visual Studio 2022 as VS developers said : " A fix for this issue has been internally implemented and is being prepared for release. We'll update you once it becomes available for download ."
Android Designer Doesn't Load After Update To VS Enterprise 2022 Ver 17.0.0 Preview 4 (64-bit) - Visual Studio Feedback

    On Tuesday, October 5, 2021, 03:53:20 PM GMT+1, Brandon Minnick ***@***.***> wrote:  
 
 


Thanks for sending the video reproduction: https://www.youtube.com/watch?v=LFQ3MkYJWpE

Is this a .NET MAUI bug or a .NET MAUI Community Toolkit bug?

**Comment by CliffAgius** [comment]
Closed as Fixed.
